docs/.vuepress/components/metadata.vue

```diff
@@ -1,43 +1,117 @@
 <template>
   <header class="metadata" v-if="!isHomePage">
-    <span class="last-updated">
-      Last updated: <time :datatime="prettyDate">{{prettyDate}}</time>
+    <span v-if="updatedSinceLastVisit" class="last-updated last-updated--message">
+      Heads up! This page has been updated since your last visit {{ daysSinceLastVisit }} days ago
```

Comment:
Can the [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat) be used and fallback to the current code?

Reply:
I don't see the added value of adding this and then also keeping the current code as fallback when this works fine.